test/unit/api-config-test.js

```diff
@@ -133,6 +133,19 @@ describe('API Config', function() {
         });
     });
 
+    describe('playbackRates', function() {
+
+        it('should round custom playback rates to nearest hundredth', function() {
+            let rates = [ 0.999, 0.6512, 0.667, 0.664 ];
+            rates = rates.filter(rate => rate >= 0.25 && rate <= 4)
+                .map(rate => Math.round(rate * 100) / 100);
```

Comment:
What happened to using the config? A test should use modules in src, not duplicate them.